packages/sitecore-jss-nextjs/src/sharedTypes/sitemap.ts

```diff
@@ -1,6 +0,0 @@
-export type StaticPath = {
```

Comment:
For me seems like it's more shared, than only related to graphql-sitemap-service, because we have also disconnected-sitemap-service

Reply:
I guess I feel like sitemap.ts is the *main* implementation, which is why I made this change. I'd be fine with changing it back though. Shall we let a 3rd person cast the decisive vote? @ambrauer 😊

Reply:
I don't mind

Reply:
I'm fine with it here.